src/backend/stream.rs

```diff
@@ -463,7 +471,8 @@ impl<'ctx> PulseStream<'ctx> {
 
             // Duplex, set up the ringbuffer
             if input_stream_params.is_some() && output_stream_params.is_some() {
-                stm.input_buffer_manager = Some(BufferManager::new(&stm.input_sample_spec))
+                let buffer_size_bytes = latency_frames * stm.input_sample_spec.frame_size() as u32;
+                stm.input_buffer_manager = Some(BufferManager::new(buffer_size_bytes as usize, &stm.input_sample_spec))
```

Comment:
The playback attribute is set as:
`tlength: buffer_size_bytes * 2,`

According to the documentation [1]: "this value should be set to the maximum latency that the application can deal with."

Here the length of the buffer is set to `latency` but we state through the buffer attributes that we can handle twice the `latency`.

Probably we need to double the size here or to use the `pa_stream_get_buffer_attr` and set the capacity of the buffer accordingly.

[1] https://freedesktop.org/software/pulseaudio/doxygen/structpa__buffer__attr.html#aa7e8f3348cbda863b6f1dd55a9024b7a